scripts/markdown.js

```diff
@@ -10,15 +10,12 @@ $.ajax({
             url: "https://api.github.com/markdown",
             data: JSON.stringify({
                 "text": data,
-                "mode": "gfm"
+                "mode": "markdown"
```

Comment:
I spent a while reading up on this earlier to be sure: `markdown` renders all the `.md` files on GitHub, while `gfm` renders user comments like [this](https://github.com/nus-cs2103/website/pull/210#issuecomment-266413313) in which (among others) headers have no `id` attributes.

> `mode`
> The rendering mode. Can be either:
> * `markdown` to render a document as plain Markdown, just like README files are rendered. 
> * `gfm` to render a document as user-content, *e.g.* like user comments or issues are rendered. In GFM mode, hard line breaks are always taken into account, and issue and user mentions are linked accordingly.
> Default: `markdown`

Source: https://developer.github.com/v3/markdown/#render-an-arbitrary-markdown-document